README.md

```diff
@@ -53,7 +53,7 @@ Alternatively, skip all the steps by using [![Gitpod Ready-to-Code](https://img.
 
 Create a file named `your-username.json` in the directory `public/data` with the following content:
 
-Optional fields: `links` and `milestones`
+Optional fields: `links`,`milestones` and `type(personal | community)`
```

Comment:
```suggestion
Optional fields: `links`, `milestones`, and `type(personal | community)`
```

Reply:
wrong. comma appears after `and`

Reply:
Superseded by https://github.com/EddieHubCommunity/LinkFree/pull/821/files#r771081622

Reply:
That's wrong according to Grammarly.

Reply:
Hmm, my Grammarly asks me to keep it after

![image](https://user-images.githubusercontent.com/91655303/146489617-42a34ea8-e391-47d5-b75f-26ddd26d8b71.png)